communication/mavlink_stream.h

```diff
@@ -90,9 +89,10 @@ typedef struct
 	byte_stream_t* rx;		///< Input stream
 	uint32_t sysid;			///< System ID
 	uint32_t compid;		///< System Component ID
-	mavlink_received_t rec;	///< Last received message
+	mavlink_received_t rec;		///< Last received message
 	bool msg_available;		///< Indicates if a new message is available and not handled yet
-	bool use_dma;			///< Indicates whether tx transfer should use dma
+	uint8_t mavlink_channel; 	///< Channel number used internally by mavlink to retrieve incomplete incoming message
```

Comment:
I don't know:
- mavlink_channel_t goes from 0 to 3 although the max number of channels is 16 on some platforms. 
- we will never read/write this value from outside the module, thus we never use the names of the enum.

Of course it would work because an enum is implemented as an int... but we would gain nothing.